include/seccomp-syscalls.h

```diff
@@ -824,6 +839,12 @@
 
 #define __SNR_getitimer			__NR_getitimer
 
+#ifdef __NR_atomic_getpagesize
+#define __SNR_getpagesize		__NR_getpagesize
+#else
+#define __SNR_getpagesize		__PNR_getpagesize
```

Comment:
I would have expected the above code to look like this:
```
#ifdef __NR_atomic_getpagesize
#define __SNR_atomic_getpagesize		__NR_atomic_getpagesize
#else
#define __SNR_atomic_getpagesize		__PNR_atomic_getpagesize
```
... is this a typo or is there some m68k cleverness going on here?

Reply:
Fixed.